tests/microservices/Microservices.DicomRelationalMapper.Tests/DicomDatasetCollectionSourceTests.cs

```diff
@@ -47,6 +55,8 @@ public void SourceReadSimpleTagToTable()
         public void TestStringTooLong(DataTooWideHandling strategy)
         {
             var ds = new DicomDataset();
+            
+            ds.AutoValidate = false;
```

Comment:
I assume this is needed to to BadMedicine using fo-dicom >= 4.0.2? This should be fine for the test packages, but we need to test the effect of the auto-validation before updating the main src.

Reply:
Agreed.  We are also currently (with this PR) building and testing on different versions of fo-dicom which is not ideal.